src/keylessBackup/SignInWithEmail.tsx

```diff
@@ -124,8 +245,15 @@ const styles = StyleSheet.create({
   scrollContainer: {
     padding: Spacing.Thick24,
   },
+  header: {
+    paddingHorizontal: variables.contentPadding,
```

Comment:
this is 16 right? seems like the scroll container uses 24, should this be consistent?

Reply:
I tried to be consistent with headers in the previous screens which appear to be 16.